packages/universal-components/src/RangeDatePicker/DraggableItem.web.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { DraggableCore, type DraggableData, type Event } from 'react-draggable';
+
+import { designTokens } from '../DesignTokens';
+import type { GrabbedSideType, DayItemSizeType } from './RangeDatePickerTypes';
+import type { OnDragEvent } from '../types';
+
+type Props = {
+  +onDrag: (OnDragEvent, GrabbedSideType) => void,
+  +onDrop: () => void,
+  +grabbedSide: GrabbedSideType,
+  +dayItemSize: DayItemSizeType,
+};
+
+export default class DraggableItem extends React.Component<Props> {
+  handleDrag = (event: Event, data: DraggableData) => {
+    this.props.onDrag(
+      {
+        nativeEvent: {
+          translationX:
+            data.lastX + data.deltaX - this.props.dayItemSize.width / 2,
+          translationY:
+            data.lastY + data.deltaY - this.props.dayItemSize.height / 2,
+        },
+      },
+      this.props.grabbedSide,
+    );
+  };
+
+  handleStop = () => {
+    this.props.onDrop();
+  };
+
+  render() {
+    return (
+      <DraggableCore onDrag={this.handleDrag} onStop={this.handleStop}>
+        <View
+          style={
+            this.props.grabbedSide === 'left'
+              ? styles.draggableItemLeft
+              : styles.draggableItemRight
+          }
+        />
+      </DraggableCore>
+    );
+  }
+}
+
+const boxStyle = {
+  width: designTokens.widthDraggableCalendarItem,
+  height: designTokens.heightCalendarItem,
+  zIndex: parseFloat(defaultTokens.zIndexOnTheTop),
+  position: 'absolute',
+};
+const styles = {
```

Comment:
While we are using `View` component we should use `StyleSheet.create(` to create styles only once. I think it will be perfect after that.

Reply:
Done